### ISSUE TITLE
Deploy BOPTEST v0.7.0 to Web-Service

### ISSUE THREAD

**Issue by dhblum** [ibpsa/project1-boptest#712]
This issue is to track deployment of BOPTEST v0.7.0 to the publicly available web-service at https://api.boptest.net.

**Comment by dhblum** [comment]
@kbenne Can you please update/close this issue when completed?

**Comment by dhblum** [comment]
@tijcolem I've tagged a v0.7.1 release via #725.  It looks like the [web](https://github.com/orgs/ibpsa/packages/container/package/boptest-web) and [worker](https://github.com/orgs/ibpsa/packages/container/package/boptest-worker) images have been published as packages with the v0.7.1 tag ok.  Please let me know if this is sufficient for you to continue deployment of the new version.  If so, please continue, and let me know when complete or any other issues.

**Comment by tijcolem** [comment]
@dhblum That's great! I'll post here when the images are pushed to production.

**Comment by tijcolem** [comment]
@dhblum  This is now pushed to prod.  https://api.boptest.net/version  I think we can close this ticket.

**Comment by dhblum** [comment]
Thank you @tijcolem!  One thing though before closing.  When I ping https://api.boptest.net/testcases I get

```
[{"testcaseid":"bestest_air"},{"testcaseid":"bestest_hydronic"},{"testcaseid":"bestest_hydronic_heat_pump"},{"testcaseid":"multizone_office_simple_air"},{"testcaseid":"multizone_residential_hydronic"},{"testcaseid":"singlezone_commercial_hydronic"},{"testcaseid":"testcase1"},{"testcaseid":"testcase2"},{"testcaseid":"testcase3"},{"testcaseid":"twozone_apartment_hydronic"}]
```
which does not include a test case that we newly released in v0.7.0 called ``multizone_office_simple_hydronic``.  I think the ``provision`` service of your production deployment needs to be updated somehow to add this new test case?

**Comment by tijcolem** [comment]
@dhblum  That's true, the update doesn't push new test cases as those are pushed by the provisioner. We can add some notes  in the updates to accommodate this though.

**Comment by tijcolem** [comment]
@dhblum  Yes, let me look over the test case deployment. I think it's just an api push to upload that test case. You can keep this ticket open until I push that.

**Comment by tijcolem** [comment]
@dhblum I uploaded the new test case multizone_office_simple_hydronic and I see it's available now when querying https://api.boptest.net/testcases.  

Did any of the existing test cases change? Should I re-upload all test cases and overwrite?

**Comment by dhblum** [comment]
Great.  Yes, other test cases did change.  It is safest to re-upload all of them and overwrite, to ensure they match what should be in the reported version of boptest.

**Comment by tijcolem** [comment]
@dhblum  Okay, I uploaded all test cases. Going forward we'll make this part of the regular update routine.

**Comment by dhblum** [comment]
That's great, thank you!  Agreed, it should be part of the regular update routine.
